### DISCUSSION TITLE
It would be nice to allow DMs

### DISCUSSION THREAD

**Discussion by glennwanjiru** [dailydotdev/daily#1338] [Feature Requests]
Connecting with other developers by direct messages would be so nice

**Comment by idoshamun** [comment]
Hi @glennwanjiru,

Thanks for the suggestion, we will consider it :)

**Comment by KristianMariyanov** [comment]
I'd also like to add my support for this feature. Right now, only a small amount of people have added their social profiles to their accounts and it's hard to reach out to someone. It would be great if this option could happen directly on the daily.dev platform.

**Comment by andradei** [comment]
Another use-case for this:

I've been posting to a Squad for a while, then the sole owner stopped approving articles, becoming the sole user to keep posting there.

There is no way to send him DMs to ask if he has any concerns about my pending posts and offering to help maintain the Squad.

A communication "channel" in that regard would be great.

So maybe not a general DM feature (as that might not be one of the goals for Daily.dev) but a way to communicate with Squad owners or inside Squads in general would help keep communities organized.

**Reply by KristianMariyanov** [reply]
More over, as a squad owners, seeing someone that post regularly, there is not an easy way to ask him if he wants to become a moderators.
